src/util/CMakeLists.txt

```diff
@@ -8,6 +8,7 @@
 ## All rights reserved.  See the file COPYING in the top-level source
 ## directory for licensing information.
 ##
+configure_file(symfpu_literal.h.in symfpu_literal.h)
 configure_file(floatingpoint.h.in floatingpoint.h)
```

Comment:
Does this still have to be  a `.h.in` file?

Reply:
Yes, for now. This will change after I refactor FloatingPointLiteral.